galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/spi/auth/UserDoc.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.framework.spi.auth;
+
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+
+public class UserDoc {
+
+    @SerializedName("_id")
+    private String userNumber;
+
+    @SerializedName("_rev")
+    private String version;
```

Comment:
`_id` and `_rev` are CouchDB-specific. Looks like this should be in the CouchDB extension rather than the framework.